packages/fluentui/react-northstar/src/components/Dropdown/Dropdown.tsx

```diff
@@ -422,6 +430,9 @@ export const Dropdown: ComponentWithAs<'div', DropdownProps> &
 
   const ElementType = getElementType(props);
   const unhandledProps = useUnhandledProps(Dropdown.handledProps, props);
+  const [unhandledA11yProps, restUnhandledProps] = partitionHTMLProps(unhandledProps, {
+    htmlProps: ['aria-labelledby', 'aria-invalid'],
+  });
```

Comment:
This should do what you want without definig custom htmlProps which is just an array in disguise :D

```suggestion
  const [unhandledA11yProps, restUnhandledProps] = partitionHTMLProps(unhandledProps, {
    includeAria: true,
  });
```

Reply:
![image](https://user-images.githubusercontent.com/22620150/103367946-9e046400-4ac6-11eb-8cbd-4be803675cba.png)

I tested it and unfortunately it doesn't help. It adds onKeyDown event into result.

Reply:
As we discussed offline, just add AriaAttributes type in a union